portal/src/assets/js/mobile/scroll-bar.js

```diff
@@ -1,16 +1,16 @@
 $(function scrollBar() {
 
 	var scrollBarWrapper = $(".scroll-bar-wrapper");
-	var mobileWrapperTxTable = $(".mobile-wrapper-tx-table");
+	var scrollBarTarget = $(".scroll-bar-target");
```

Comment:
I don't think changing `mobile-wrapper-tx-table` to `.scroll-bar-target` is good idea. It will make the page what is using `mobile-wrapper-tx-table` break. The `block detail` page is using `mobile-wrapper-tx-table`. I just tested, the scroll bar on the `block detail` page is not working now. I think the easiest way to fix is creating a new scroll bar instead of renaming the old one.
![image](https://user-images.githubusercontent.com/14008461/69693103-2d5b7d00-1139-11ea-8079-9b57e440f6ce.png)

Reply:
Should be working now, i didn't attach the `.scroll-bar-target` class to the target element. Using a generic name for the element the scrollbar is targeting makes development much easier.

Reply:
Thank you for the update. Yeah, it's working now, but the scroll-bar on the home page is using `mobile-wrapper-tx-table` as well and it breaks as well. Also, there are some css codes related to `mobile-wrapper-tx-table`. May I ask the reason you rename `mobile-wrapper-tx-table` to `.scroll-bar-target`?

Reply:
Made the change